apps/monitoring/acr-sync/check-acr-sync.yaml

```diff
@@ -1,50 +1,61 @@
-apiVersion: batch/v1
-kind: CronJob
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
 metadata:
   name: check-acr-sync
   namespace: monitoring
   labels:
     app: check-acr-sync
 spec:
-  schedule: "*/15 * * * *"
-  concurrencyPolicy: "Forbid"
-  failedJobsHistoryLimit: 3
-  startingDeadlineSeconds: 300
-  jobTemplate:
+  chart:
     spec:
-      backoffLimit: 3
-      template:
+      chart: cronjob-chart
+      sourceRef:
+        kind: GitRepository
+        name: chart-repo
+        namespace: flux-system
+      interval: 1m
+  interval: 5m
+  values:
+    cronJob:
+      schedule: "*/15 * * * *"
+      concurrencyPolicy: "Forbid"
+      failedJobsHistoryLimit: 3
+      startingDeadlineSeconds: 300
+      jobTemplate:
```

Comment:
Should `jobTemplate` be in this file if its converting to a helmRelease chart-job type?